demo/src/main/java/com/bluelinelabs/conductor/demo/MainActivity.java

```diff
@@ -12,22 +12,20 @@
 
 import butterknife.BindView;
 import butterknife.ButterKnife;
-import butterknife.Unbinder;
 
-public class MainActivity extends AppCompatActivity implements ActionBarProvider {
+public final class MainActivity extends AppCompatActivity implements ActionBarProvider {
 
     @BindView(R.id.toolbar) Toolbar toolbar;
     @BindView(R.id.controller_container) ViewGroup container;
 
     private Router router;
-    private Unbinder unbinder;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
 
         setContentView(R.layout.activity_main);
-        unbinder = ButterKnife.bind(this);
+        ButterKnife.bind(this);
```

Comment:
All references to the activity (and its views) are released after onDestory(). The unbinder is not required.